packages/commons/src/config/consumerServiceConfig.ts

```diff
@@ -9,10 +9,12 @@ export const KafkaConsumerConfig = KafkaConfig.and(
       TOPIC_STARTING_OFFSET: z
         .union([z.literal("earliest"), z.literal("latest")])
         .default("latest"),
+      RESET_CONSUMER_OFFSETS: z.string().default("false"),
```

Comment:
How about parsing this directly as a boolean?

Reply:
Env vars in `process.env` are always read as strings, if I define it as `z.boolean()` here I get an error:

<img width="403" alt="image" src="https://github.com/user-attachments/assets/40aee145-8e3c-4a26-83f7-ffbd09ffc941">

We could coerce to string with `z.coerce.boolean()`, but at the moment we never do it around the app for env variables and config definitions, could be worth tracking as a potential improvement

Reply:
You're absolutely right, let's leave it like this and track this as potential improvement on the board